addons/crm/data/mail_template_data.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <!-- TDE note: this should not be a mail template -->
+        <record id="mail_template_crm_access_request" model="mail.template">
+            <field name="name">Mail: Access Request</field>
+            <field name="model_id" ref="crm.model_crm_module_access_request"/>
+            <field name="subject">Module Access Request for "{{ object.module_id.shortdesc }}"</field>
+            <field name="email_from">{{ object.user_id.email_formatted or user.email_formatted }}</field>
+            <field name="partner_to" >{{ ctx.get('partner', {}).id }}</field>
```

Comment:
It means you template is basically not working, hence: either you have a way to store the partner, either it should be a technical template (aka: a web view). Otherwise this template is mainly not usable in standard flows, and trying to edit it is going to break it. Imagine a standard user updating it, they won't know it heavily relies on context :) .